test/functional/DDR_Test/tck_ddrext.xml

```diff
@@ -186,12 +186,14 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<echo>Using JVM : ${sdk.executable}${sdk.executable.ext}</echo>
 		<echo>classname = "j9vm.test.corehelper.StackMapCoreGenerator"</echo>
 		<echo>Java VM Args:</echo>
-		<echo>  jvmarg = -Xshareclasses:name=ddrextjunitSCC,reset -Xitsn1000</echo>
+		<echo>  jvmarg = -Xshareclasses:name=ddrextjunitSCC,reset -Xitsn1000 -Xjit:disableAsyncCompilation -Xmx512M</echo>
```

Comment:
This should be `-Xaot` instead of `-Xjit`

Reply:
I've added `-Xaot:disableAsyncCompilation`, but for whatever reason, the test fails without `-Xjit:disableAsyncCompilation` (https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder/15744/console)

Reply:
Could that be why count=5 doesn't work, because the test is setting `-Xaot:disableAsyncCompilation` instead of `-Xaot:disableAsyncCompilation`? @dsouzai

Reply:
No, putting that specific option on the `-Xjit` cmd line is the right thing because of https://github.com/eclipse-openj9/openj9/blob/6d89b11148c84ce64700aa73e3d1d1631ab59cbd/runtime/compiler/control/CompilationThread.cpp#L1292
hilariously, if it was to be put on `-Xaot` it wouldn't be respected 🙄

Reply:
So we should try count=5 with `-Xjit:disableAsyncCompilation`

Reply:
I thought @midronij tried that and it didn't help?

Reply:
That's correct, `-Xjit:disableAsyncCompilation` with count=5 (or any count value that isn't 0) doesn't work